src/index.html

```diff
@@ -3,10 +3,403 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <!-- <link rel="stylesheet" href="./styles/style.css"> -->
```

Comment:
There is the commented out link to the stylesheet file. If it is not needed, it should be removed from the code.